modules/core/src/stat.cpp

```diff
@@ -2282,10 +2278,10 @@ static bool ipp_minMaxIdx( Mat &src, double* minVal, double* maxVal, int* minIdx
                 depth == CV_8S ? (ippiMinMaxIndxFuncC1)ippiMinMaxIndx_8s_C1R :
 #endif
                 depth == CV_16U ? (ippiMinMaxIndxFuncC1)ippiMinMaxIndx_16u_C1R :
-#if !((defined _MSC_VER && defined _M_IX86) || defined __i386__)
+#if IPP_DISABLE_BLOCK
```

Comment:
We observed issues here on 32-bit builds. Tests for x64 builds were fine on AMD CPUs.

What is about this condition?
```
#if !((defined _MSC_VER && defined _M_IX86) || defined __i386__) || IPP_DISABLE_BLOCK
```